overlord/auth/auth.go

```diff
@@ -208,8 +209,30 @@ func LoginCaveatID(m *macaroon.Macaroon) (string, error) {
 func (ma *MacaroonAuthenticator) Authenticate(r *http.Request) {
 	var buf bytes.Buffer
 	fmt.Fprintf(&buf, `Macaroon root="%s"`, ma.Macaroon)
```

Comment:
In-case you're not aware (apologies if teaching to suck eggs) but Go fmt uses %q to automatically pretty print things, so root=%q would put the quotes around for you.